src/main/java/ee/carlrobert/codegpt/settings/service/you/YouSettings.java

```diff
@@ -20,6 +21,9 @@ public YouSettingsState getState() {
   @Override
   public void loadState(@NotNull YouSettingsState state) {
     this.state = state;
+    if (this.state.getChatMode() == null) {
+      this.state.setChatMode(YouCompletionMode.DEFAULT);
```

Comment:
I think this is unnecessary, you can set it already as the default value. It shouldn't cause any problems since it's a new field.

Reply:
I'm not a java developer, sorry. Can you give me an example of how to do this?

ChatMode and Model aren't on the setup form, so they aren't stored in the xml, but the mode should be initialized into something not-null. that's why this code is here

Reply:
```java
private boolean useGPT4Model;
private YouCompletionMode chatMode = YouCompletionMode.DEFAULT;
private YouCompletionCustomModel customModel;
```

https://github.com/carlrobertoh/CodeGPT/pull/399/files#diff-def42ea7a06eeb949bd57cd4152c41ab0d8b887c8d95fac817a95130f07d63d0R12